lib/assets/i18n/log/messages_en.json

```diff
@@ -77,24 +77,12 @@
   "ZWED0094I":"Extracted war to %s",
   "ZWED0095I":"Making junction from %s to %s",
   "ZWED0096I":"Making symlink from %s to %s",
-  "ZWED0097I":"%s Websocket closed. Total remaining terminals connected: %s",
-  "ZWED0098I":"Total TN3270 terminals connected: %d",
-  "ZWED0099I":"Total TN5250 terminals connected: %d",
-  "ZWED0100I":"Total VT terminals connected: %d",
-  "ZWED0101I":"%s Connected. Total terminals connected: %d",
-  "ZWED0102I":"Total TN3270 terminals connected: %d",
-  "ZWED0103I":"Total TN5250 terminals connected: %d",
-  "ZWED0104I":"Total VT terminals connected: %d",
-  "ZWED0105I":"Found and loaded compatible handler file /lib/%s",
-  "ZWED0106I":"Saw Websocket request, method=%s",
-  "ZWED0107I":"Saw Websocket request, method=%s",
-  "ZWED0108I":"Saw Websocket request, method=%s",
   "ZWED0109I":"Registering App (ID=%s) with App Server",
   "ZWED0110I":"App %s installed to %s and registered with App Server",
   "ZWED0111I":"Authentication plugin %s added to category %s",
   "ZWED0112I":"Auth enabled=false. Auth passthrough.",
   "ZWED0113I":"%s",
-  "ZWED0114I":"adding plugin remotely %s",
+  "ZWED0114I":"adding plugin remotely ",
```

Comment:
Let's capitalize this for consistency's sake

Reply:
addressed